src/Servers/IIS/IIS/samples/NativeIISSample/Startup.cs

```diff
@@ -117,14 +118,22 @@ public void Configure(IApplicationBuilder app)
 
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseIIS()
-                .UseIISIntegration()
-                .UseStartup<Startup>()
-                .Build();
-
-            host.Run();
+            AppDomain.CurrentDomain.FirstChanceException += (sender, eventArgs) => {
```

Comment:
Had to use `FirstChanceException` instead of `UnhandledException` because we don't have https://github.com/dotnet/coreclr/pull/23054 yet.